### PR TITLE
Add Google Chrome Platform Notifications

### PR DESCRIPTION
**Pull request by sydp**
Adds a definition for the Google Platform Notifications leveldb

### CODE REVIEW

File: data/webbrowser.yaml [hunk 1]
```diff
@@ -856,7 +856,7 @@ sources:
 supported_os: [Darwin, Linux, Windows]
 urls: ['https://forensics.wiki/google_chrome#configuration']
 ---
-name: ChromeNotifications
+name: ChromePlatformNotifications
```
Comment (joachimmetz) [inline data/webbrowser.yaml]: Please keep `ChromeNotifications` as an alias. Also maybe `ChromePlatformNotificationsDatabase` ?

Comment (sydp) [reply data/webbrowser.yaml]: Done

File: data/webbrowser.yaml [hunk 2]
```diff
@@ -816,6 +816,32 @@ sources:
   supported_os: [Windows]
 supported_os: [Darwin, Linux, Windows]
 ---
+name: ChromePlatformNotifications
+aliases: [ChromeNotifications, ChromePlatformNotificationsDatabase]
+doc: |
+  Google Chrome Platform Notifications LevelDB.
+
+  The Platform Notifications directory contains the files that comprise a LevelDB
+  database, which in turn holds platform notificaiton data.
```
Comment (joachimmetz) [inline data/webbrowser.yaml:825]: notificaiton => notification

File: data/webbrowser.yaml [hunk 3]
```diff
@@ -816,6 +816,32 @@ sources:
...
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/Library/Application Support/Google/Chrome/*/Platform Notifications/*'
```
Comment (joachimmetz) [inline data/webbrowser.yaml:830]: use `[ ... ]` notation per style guide

File: data/webbrowser.yaml (hunk 2 above)
Comment (sydp) [reply data/webbrowser.yaml:825]: Done

File: data/webbrowser.yaml (hunk 3 above)
Comment (sydp) [reply data/webbrowser.yaml:830]: Done

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**